frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -146,7 +145,7 @@ function FileManagerHistory:onMenuHold(item)
             self._manager.statuses_fetched = false
         end
         self._manager:updateItemTable()
-        self._manager.files_updated = true -- sidecar folder may be created/deleted
+        self._manager.files_updated = true
```

Comment:
(Why remove this comment ? It is no longer true ?)

Reply:
Because refresh is required for other reasons too (eg status was changed).